src/ui/components/Timeline.js

```diff
@@ -167,11 +167,9 @@ class WebReplayPlayer extends Component {
       this.onUnprocessedRegions.bind(this)
     );
 
-    this.console.on("ready", () => {
-      const consoleFrame = this.console.hud.ui;
-      consoleFrame.on("message-hover", this.onConsoleMessageHover);
-      consoleFrame.wrapper.subscribeToStore(this.onConsoleUpdate);
-    });
+    const consoleFrame = this.console.hud.ui;
+    consoleFrame.on("message-hover", this.onConsoleMessageHover);
+    consoleFrame.wrapper.subscribeToStore(this.onConsoleUpdate);
```

Comment:
the console is now loaded before we show the timeline, which should be tweaked for performance, but this was already getting big